src/main/webapp/jsp/instructorFeedbackResultsByQuestion.jsp

```diff
@@ -9,6 +9,6 @@
 <results:resultsPage pageTitle="TEAMMATES - Feedback Session Results" bodyTitle="Session Results" jsIncludes="${jsIncludes}">
     <%-- Responses are displayed when only a single question is loaded, 
          otherwise it is loaded through ajax. See /js/instructorFeedbackResultsAjaxByQuestion.js --%>
-    <results:byQuestionResults isShowingResponses="${!data.largeNumberOfRespondents}"
+    <results:byQuestionResults isShowingResponses="${data.largeNumberOfRespondents}"
```

Comment:
Do some investigation what the intention of `data.largeNumberOfRespondents` was. My guess is that it's legacy code superseded by the commit that stopped all results from auto-loading. If it is, this can be replaced with `isShowingResponses="${false}"`. But check first.